File: src/main/java/com/designpatterns/dev/behavioral/visitor/java/operation/TextHinting.java
package com.designpatterns.dev.behavioral.visitor.java.operation;

import com.designpatterns.dev.behavioral.visitor.java.HtmlUIElement;

public class TextHinting implements Operations {
    @Override
    public String apply(HtmlUIElement htmlUIElement) {
        return "Text Hinting Feature for " + htmlUIElement.getClass().getName();
    }
}

File: src/main/java/com/designpatterns/dev/behavioral/visitor/kotlin/operation/TextHinting.kt
package com.designpatterns.dev.behavioral.visitor.kotlin.operation

import com.designpatterns.dev.behavioral.visitor.kotlin.HTMLElement

class TextHinting : Operation {
    override fun apply(htmlElement: HTMLElement): String {
        return "Text Hinting Feature for " + htmlElement.javaClass.name
    }
}